### ISSUE TITLE
Incomplete error message on disk without DIO support

### ISSUE THREAD

**Issue by turbo** [bloomberg/comdb2#831]
So, this happened:

```text
➜ comdb2 --create --dir /media/dbdsk/db testdb2
[ERROR] Can't open/create /media/dbdsk/db/watchdog: 22 Invalid argument
[FATAL] Direct IO is not supported for dir /media/dbdsk/db
Please set 'setattr directio 0' in lrl

➜ echo 'setattr directio 0' >> /media/dbdsk/db/testdb2.lrl 

➜ comdb2 --dir /media/dbdsk/db testdb2           
Non-registered tunable 'setattr'.
[FATAL] ZERO log files found in /media/dbdsk/db/logs!
[FATAL] Cannot start without logfiles.  If this is
```

The error message ends at "If this is", so I can't really figure out where to go from here. How do I get the database up on this disk (which is a `tmpfs`)?

**Comment by akshatsikarwar** [comment]
Something along the following lines should do it
```
mkdir /media/dbdsk/db/testdb2
cat > /media/dbdsk/db/testdb2/testdb2.lrl <<EOF
name testdb2
dir /media/dbdsk/db/testdb2
setattr directio 0
EOF
comdb2 testdb2 --create --lrl  /media/dbdsk/db/testdb2/testdb2.lrl
```

**Comment by adizaimi** [comment]
We do exactly what you want in the tests/setup, first we create the lrl with that option, then run create.
All you need to do now (after the steps you did above) is the following create command:
```
  comdb2 --create --lrl /media/dbdsk/db/testdb2.lrl testdb2
```
and then you can run it with ```comdb2 --dir /media/dbdsk/db testdb2```

**Comment by turbo** [comment]
Yep, that works. Still, the bug with the error message being cut off remains.

**Comment by adizaimi** [comment]
Thanks for pointing it out--it's an error msg that only the first half is printed at the `FATAL` level and the rest is at `INFO` level thus it does not get flushed to the user's console. Will fix. Thanks.

**Comment by akshatsikarwar** [comment]
Fixed: 4913048280538beb2df4ae651b5dd75f8f5ab92e
